guide/glossary.md

```diff
@@ -241,46 +294,39 @@ A small piece of data that has all the information needed to generate a specific
 
-## Public key
-A bitcoin address' public key can be derived from the private key. The address itself is a hash of the public key.
+Many wallet-applications work with HD Wallets and recovery phrases, and are interoperable, meaning you can change the application that can control your wallet should you wish (although there are some caveats depending on if they support just BIP32 or also BIP44).
 
-## Replace-by-fee (RBF)
-A node policy that allows an unconfirmed transaction to be replaced with a different transaction that spends at least one of the same inputs and which pays a higher transaction fee. This can sometimes be useful if a transaction has got *stuck* during times of higher-than-normal network-fees.
+### Remote procedure calls (RPC)
 
-## Recovery phrase
+A remote procedure calls (RPC) is when a command is sent to a remote system where it's processed and results are returned. Bitcoin uses RPC to connect to running instances of bitcoind and issue commands such as sending transactions or querying the Bitcoin block chain.
 
-_Also referred to as Seed, Mnemonic, and Backup phrase._
+- [Bitcoin Core RPC docs](https://bitcoincore.org/en/doc/)
 
-The controlling keypair of a bitcoin wallet can be derived from a *recovery phrase* of 12 words (or 18 or 24, which is less common) from a standardized list, defined in [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki). The recovery phrase provides full access to a bitcoin wallet as it contains the private key and is therefore very valuable. It’s extremely important to keep it safe, both from other people getting access to it and for yourself not to lose it by creating one or several backups of the phrase. In many applications most of this work falls on the user and it’s important to acknowledge the responsibility here of the makers of the application to ensure that the user is able and aware of how to securely store a recovery phrase backup.
+### Replace-by-fee (RBF)
+A node policy that allows an unconfirmed transaction to be replaced with a different transaction that spends at least one of the same inputs and which pays a higher transaction fee. This can sometimes be useful if a transaction has got *stuck* during times of higher-than-normal network-fees.
 
-Many wallet-applications work with HD Wallets and recovery phrases, and are interoperable, meaning you can change the application that can control your wallet should you wish (although there are some caveats depending on if they support just BIP32 or also BIP44).
 
 **Technicalities** - Recovery of multisig-wallets needs both the extended public key and the recovery phrase of all paticipating keys as well as the master key fingerprint as defined by BIP32 concatenated with the derivation path of the public key. The derivation path is represented as 32-bit little endian unsigned integer indexes concatenated with each other. The number of 32 bit unsigned integer indexes must match the depth provided in the extended public key.
```

Comment:
Is this technicalities copy in the right place now?

Reply:
No it is not, will rebase - personally I'd just remove this. I'm more for simple details in the glossary and link to more technical explanations externally or link to sections in the Guide that put the term in context like this.